### DISCUSSION TITLE
Group collaboration

### DISCUSSION THREAD

**Discussion by Yiannis1962** [te-online/timemanager#190] [Q&A]
Is there a way that all members of a shared group are able to see all entries ?
Currently it is only me as admin who created a new client, who is able to see all time entries from users of the group.

**Comment by te-online** [comment]
Hi @Yiannis1962 I'm afraid that is the intended use-case. The admin can see all time entries, sharees can only see their own time entries. Sorry!

**Reply by altrasoluzione** [reply]
Hi,
It would be very useful to configure the permissions of each shared user. For example, it would be useful to allow a client to see the report of all the activities carried out on their projects but not allow them to make changes or additions—the exact opposite of what happens now for users with whom the client is shared.

Anyway, great App! Simple but useful! Thanks!

**Reply by te-online** [reply]
I agree, it would be useful. Unfortunately it is a major task to make it happen 😿

**Reply by altrasoluzione** [reply]
😢
